Tests/Common.ps1

```diff
@@ -33,29 +33,38 @@ function Initialize-CommonTestSetup
     param()
 
     $moduleRootPath = Split-Path -Path $PSScriptRoot -Parent
-    . (Join-Path -Path $moduleRootPath -ChildPath 'Tests\Config\Settings.ps1')
+    $settingsPath = Join-Path -Path $moduleRootPath -ChildPath 'Tests\Config\Settings.ps1'
```

Comment:
```suggestion
    $settingsPath = Join-Path -Path $moduleRootPath -ChildPath 'Tests/Config/Settings.ps1'
```

We should have been using forward slash to be more universal.  Not your fault, but want to fix this instead of propagating the mistake further.

Reply:
You're right. Can do.